src/App.js

```diff
@@ -45,23 +45,26 @@ class App extends Component {
     .catch(error => console.log(error))
   }
 
-  toggleLandingScreen() {
+  toggleLandingScreen = () => {
```

Comment:
we should probably go back and make sure that all of our custom methods are arrow functions. i'll add an issue.